model_analyzer/record/metrics_manager.py

```diff
@@ -246,7 +246,7 @@ def profile_models(self, run_config: RunConfig) -> Optional[RunConfigMeasurement
             else FileWriter(self._config.perf_output_path)
         )
         capture_gpu_metrics = (
-            run_config.cpu_only() and not self._config.always_report_gpu_metrics
+            self._config.always_report_gpu_metrics or not run_config.cpu_only()
```

Comment:
Can we add unit testing?

Reply:
Not easily. Metrics manager is not covered by unit testing.